Add management-api documentation (#228)
wso2/micro-integrator: doc/working-with-the-micro-integrator.md

</registry>
```

## Micro Integrator CLI

Micro Integrator CLI is a command line tool for managing the WSO2 Micro Integrator. You can retrieve information about the artifacts deployed in the Micro Integrator with this tool.

> Please refer to the instructions on the configuration of, and working with, the Micro Integrator CLI tool by visiting the documentation [Working with the CLI](../cmd/README.md)

Reviewer: ```suggestion
Please refer to the instructions on working with the Micro Integrator CLI tool by visiting the documentation [Working with the CLI](../cmd/README.md)
```

> ### Micro Integrator Management Api

Reviewer: ```suggestion
### Micro Integrator REST Api
```

> The Micro Integrator Management Api is the way to get data about the deployed artifacts. It's an HTTP-based API that can use to query data about the artifacts. More details can be found in the [Management Api Documentation](./management-api.md)

Reviewer: ```suggestion
The Micro Integrator REST API can be used to get data about the deployed artifacts. It is an HTTP based API that can use to query data about the artifacts. More details can be found in the [REST API Documentation](./management-api.md)
```
